### ISSUE TITLE
Error at compilation

### ISSUE THREAD

**Issue by FranckSix** [telerik/kendo-themes#4330]
**Describe the bug**
Files :
./scss/core/functions/index.import.scss _import ./scss/functions/index.import.scss_ **Error: File to import not found or unreadable**
./scss/core/color-system/index.import.scss _import ./scss/color-system/index.import.scss_ **Error: File to import not found or unreadable**
./scss/core/mixins/index.import.scss _import ./scss/mixins/index.import.scss_ **Error: File to import not found or unreadable**
./scss/core/module-system/index.import.scss _import ./scss/module-system/index.import.scss_ **Error: File to import not found or unreadable**
./scss/core/styles/index.import.scss _import ./scss/styles/index.import.scss_ **Error: File to import not found or unreadable**

**To reproduce**
under the ./scss folder unable to find the referenced import folders

None of theses folders are found : ./scss/functions, ./scss/color-system, ./scss/mixins, ./scss/module-system, ./scss/styles

**Affected package (please remove the unneeded items)**
* theme-default

**Affected suites (please remove the unneeded items)**
* Kendo UI for jQuery

**Affected browsers (please remove the unneeded items)**
* Not Applicable

**Build system information (please remove the unneeded items)**
* OS: Windows 11
* Node version: 19.7.0
* NPM version: 9.5.0

**Additional context**

> {
>   "status": 1,
>   "file": "D:/Projets/node_modules/@progress/kendo-theme-default/scss/core/functions/index.import.scss",
>   "line": 1,
>   "column": 1,
>   "message": "File to import not found or unreadable: @progress/kendo-theme-core/scss/functions/index.import.scss.",
>   "formatted": "Error: File to import not found or unreadable: @progress/kendo-theme-core/scss/functions/index.import.scss.\n        on line 1 of node_modules/@progress/kendo-theme-default/scss/core/functions/index.import.scss\n        from line 4 of node_modules/@progress/kendo-theme-default/scss/core/_index.scss\n        from line 1 of node_modules/@progress/kendo-theme-default/scss/button/_index.scss\n        from line 23 of helios/Dev/Dev/Src/Helios/Helios.UI/Content/Kendo/KendoStyle.scss\n        from line 31 of helios/Dev/Dev/Src/Helios/Helios.UI/Content/Kendo/KendoStyleLight.scss\n>> @import \"@progress/kendo-theme-core/scss/functions/index.import.scss\";\n\n   ^\n"
> }

**Comment by Juveniel** [comment]
@FranckSix could you please add additional information on how are you building the theme, e.g what build tool ( webpack, sass, etc ) and how the theme is imported in the `scss` file ?

**Comment by FranckSix** [comment]
I change on all index-import under core folder

@import "@progress/kendo-theme-core/scss/functions/index.import.scss";
to
@import "node_modules/@progress/kendo-theme-core/scss/functions/index.import.scss";

now its working

**Comment by joneff** [comment]
@FranckSix can you paste here your JS file with the setup? I tested locally with node-sass and things worked on my end. Of course, it might be something I am missing.

Previously, we shipped the themes with all the dependencies inlined i.e. included in the package. That was not ideal, so we had a couple of releases in which we used `~` to point to node_modules imports. But then, you now dependencies like webpack warned about not using `~` and using includePaths / loadPaths instead, so we did.

Long story short, to correctly compile the themes, when it's not the full theme, you need to instruct the compiler where to look for files it cannot find. The configuration for node-sass is called includePaths -- https://sass-lang.com/documentation/js-api/interfaces/LegacyFileOptions#includePaths.

Still, I would like to see the code you use to compile so I can test it locally (for guidance and bugs).

**Comment by Juveniel** [comment]
Closing this issue, as the points that @joneff outlined resolve it successfully.
